src/commands/test/cheatcodes/cheat_sources/cheats.cairo

```diff
@@ -1,22 +1,44 @@
 %lang starknet
 
-
-func roll(blk_number: felt):
-   %{ syscall_handler.set_block_number(ids.blk_number) %}
-   return ()
+func roll(blk_number : felt):
+    %{ syscall_handler.set_block_number(ids.blk_number) %}
+    return ()
 end
 
-func warp(blk_timestamp: felt):
-   %{ syscall_handler.set_block_timestamp(ids.blk_timestamp) %}
-   return ()
+func warp(blk_timestamp : felt):
+    %{ syscall_handler.set_block_timestamp(ids.blk_timestamp) %}
+    return ()
 end
 
-func start_prank(caller_address: felt):
+func start_prank(caller_address : felt):
     %{ syscall_handler.set_caller_address(ids.caller_address) %}
     return ()
 end
 
 func stop_prank():
     %{ syscall_handler.set_caller_address(None) %}
     return ()
-end
\ No newline at end of file
+end
+
+func mock_call(contract_address : felt):
+    %{
+        if mocked_fn_name is None:
```

Comment:
If variable is not defined this will throw. 
```
>>> dsad is None
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
NameError: name 'dsad' is not defined
```
https://stackoverflow.com/questions/1592565/determine-if-variable-is-defined-in-python